Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -415,15 +415,15 @@ function New-LDAPSIdentitySource {
         }
         else {
             Write-Warning "$($ExternalIdentitySources | Format-List | Out-String)"
-            Write-Warning "Identity source already exists, but has a different name. Continuing..."
+            Write-Warning "An Identity source already exists, but it's different. Continuing to add this one..."
```

Comment:
The stream here is debatable as the customer could desire multiple identity sources with different names. This could be Info stream vs. warning.

Reply:
Also see comment on same message for LDAP cmdlet